include/OpenSpaceToolkit/Astrodynamics/Flight/Profile/Model/Tabulated.hpp

```diff
@@ -122,7 +118,7 @@ class Tabulated : public virtual Model
     ///
     /// @param aFrameName Name of the frame
     /// @return Shared pointer to the frame
-    virtual Shared<const Frame> getBodyFrame(const String& aFrameName) const override;
+    virtual Shared<const Frame> constructBodyFrame(const String& aFrameName) const override;
```

Comment:
This is just a rename right? Technically not part of the scope of this MR for removing only the depracated stuff. Can we split these renames into a separate MR? It will be important to be disciplined with the scope of the breaking MRs we merge into the devel branch to minimize rebasing pain